examples/allennlp/classifier.jsonnet

```diff
@@ -0,0 +1,72 @@
+
+
+// use dev.jsonl for training to reduce computation time.
+local TRAIN_PATH =  "https://s3-us-west-2.amazonaws.com/allennlp/datasets/imdb/dev.jsonl";
+local VALIDATION_PATH =  "https://s3-us-west-2.amazonaws.com/allennlp/datasets/imdb/test.jsonl";
+
+local DROPOUT = std.extVar("DROPOUT");
+local EMBEDDING_DIM = std.extVar("EMBEDDING_DIM");
+local CNN_FIELDS(max_filter_size, embedding_dim, hidden_size, num_filters) = {
+  "type": "cnn",
+  "ngram_filter_sizes": std.range(1, max_filter_size),
+  "num_filters": num_filters,
+  "embedding_dim": embedding_dim,
+  "output_dim": hidden_size,
+};
+local ENCODER = CNN_FIELDS(
+    std.parseInt(std.extVar("MAX_FILTER_SIZE")),
+    EMBEDDING_DIM,
+    std.extVar("HIDDEN_SIZE"),
+    std.extVar("NUM_FILTERS")
+);
+
+
+{
+  "numpy_seed": 42,
+    "pytorch_seed": 42,
```

Comment:
`numpy_seed` and `pytorch_seed` seem to be in the same nest-level, but they have different indent-levels.

Reply:
Thank you for pointing out, I run `jsonnetfmt` in 105cbcd.